hydradx/model/plot_utils.py

```diff
@@ -1,224 +1,27 @@
-import matplotlib.pyplot as plt
+import matplotlib.pyplot as plot
+import re
 
 
-def plot_vars(df, var_list: list, sim_labels: list = ['0', '1']) -> None:
+def plot_vars(df, var_list: list, sim_labels: list = ('0', '1')) -> None:
     simulations = df.simulation.unique()
     print(simulations)
     for var in var_list:
-        fig = plt.figure(figsize=(20, 5))
+        fig = plot.figure(figsize=(20, 5))
         if var in df.columns:
+            # rows, columns, index
             bounds = (1, 3, 1)
-            ax = plt.subplot(*bounds, title=var)
+            ax = plot.subplot(*bounds, title=var)
             for i in simulations:
                 df[[var, 'timestep']][df['simulation'] == i].astype(float).plot(ax=ax, y=[var], x='timestep',
                                                                                 label=[sim_labels[i]])
-        elif var + '-0' in df.columns:
-            max_i = 0
-            while var + '-' + str(max_i + 1) in df.columns:
-                max_i += 1
-            for i in range(max_i + 1):
-                # rows, columns, index
-                bounds = (1, max_i+1, i+1)
-                var_i = var + '-' + str(i)
-                ax = plt.subplot(*bounds, title=var_i)
-                for j in simulations:
-                    df[[var_i, 'timestep']][df['simulation'] == j].astype(float).plot(ax=ax, y=[var_i], x='timestep',
-                                                                                      label=[sim_labels[j]])
-    plt.show()
-
-
-'''
-
-def plot_var_list(var_list, labels, rdf, run=1):
```

Comment:
did a bunch of functions get removed because they don't happen to be getting used in the notebooks currently checked in?

Reply:
They were all commented out and had been since I first looked at this code, so I assumed they had been deprecated.

Reply:
Ah, okay. Can we delete them then?

Reply:
Already did